CONTRIBUTING.md

```diff
@@ -0,0 +1,206 @@
+# Contributing Guide
+
+Any contribution to this project means implicitly that you accept the
+[code of conduct](CODE_OF_CONDUCT.md) from this project.
+
+See [Build system](#build-system) for a reference of the build system.
+
+## Requirements
+
+- [Git][]
+- [Go][] >= 1.21
+
+[Git]: https://git-scm.com/
+[Go]: https://golang.org/dl/
+
+**Optional:**
+
+- [GolangCI Lint][] >= 1.54
+- [GNU Make][] >= 4.3 (build tool)
+- [reflex][] >= 0.3 (filesystem watching)
+
+[GolangCI Lint]: https://github.com/golangci/golangci-lint/releases
+[GNU Make]: https://www.gnu.org/software/make/
+[reflex]: https://github.com/cespare/reflex
```

Comment:
As **reflex** is a Go library, would you mind if we add it to the `go.mod` file (along with any other necessary packages) so that it gets installed automatically when running `$ go mod tidy`?

Reply:
Upon further reading the PR, I understood that this is not necessary.